djstripe/templates/djstripe/confirm_form.html

```diff
@@ -19,10 +19,7 @@
 {% endif %}
 
 <div class="row">
-    {% for plan in PLAN_LIST %}
-      {% with plan_count=PLAN_LIST|length %}
-        <div class="col-xs-{{ 12|djdiv:plan_count|floatformat }}">
-      {% endwith %}
+        <div class="col-xs-12">
             <form
               {% if not customer.current_subscription or customer.current_subscription.status == CurrentSubscription.STATUS_CANCELLED %}
                   action="{% url 'djstripe:subscribe' %}" class="djstripe-subscribe"  
```

Comment:
here you will need to change action as follow:

    action="{% url 'djstripe:confirm' plan=plan.plan %}" class="djstripe-subscribe"